tslint.json

```diff
@@ -60,7 +60,7 @@
         "chai-prefer-contains-to-index-of": true,
         "chai-vague-errors": true,
         "encoding": true,
-        "export-name": true,
+        "export-name": [true, { "allow": ["^Rule$"] }],
```

Comment:
Quick fix instead of disabling/enabling check in 80 files, which is better for separate PR.